rdf_differ/skos_history_wrapper.py

```diff
@@ -32,37 +32,43 @@
 INPUT_MIME_TYPE=\"{input_type}\""""
 
 
+class SKOSException(Exception):
```

Comment:
I would call that `SkosHistoryRunnerException`, in case we need it, but maybe we don't.